scripts/teamDataScript.js

```diff
@@ -316,7 +316,7 @@ function loadMatch()
     var cname = `/season_config/${season}/` //starter bit of season_config_name
     var cookieList = document.cookie.split(';');
 
-    var matchInputHTML = `<input type="number" class="matchNumInput match-frequency js_clear_on_load">`;
+    var matchInputHTML = `<input style = "width: 80%" type="number" class="matchNumInput match-frequency js_clear_on_load">`;
```

Comment:
this works great - apparently this is how you need to do it for the pit answers too.